test/app/CMakeLists.txt

```diff
@@ -83,13 +89,13 @@ CHECK_BUILD_DEPENDENCY_AGAINST_LIST(
 
 
 LIST(APPEND CMAKE_PREFIX_PATH "${DEPENDENCY_INSTALL_DIR}")
-FIND_PACKAGE(shared_library REQUIRED)
+FIND_PACKAGE(shared-library-shared REQUIRED)
 
-ADD_LIBRARY(shared_library_for_prerun-shared SHARED IMPORTED)
-SET_TARGET_PROPERTIES(shared_library_for_prerun-shared
+ADD_LIBRARY(shared-library-for-prerun-shared SHARED IMPORTED)
```

Comment:
Maybe remove prefix shared- to remove duploicity

Reply:
Yes but I would need to rename it on all places....